Toastify/src/Core/ApplicationStartupException.cs

```diff
@@ -5,44 +5,42 @@ namespace Toastify.Core
 {
     public class ApplicationStartupException : ApplicationException
     {
-        public ApplicationStartupException() : this(true)
+        public ApplicationStartupException(string message) : base(message + Environment.NewLine + CreateMessage())
         {
         }
 
-        public ApplicationStartupException(bool spotifyStatus) : base(CreateMessage(spotifyStatus))
+        private static string CreateMessage()
         {
-        }
-
-        public ApplicationStartupException(string message) : this(message, true)
-        {
-        }
+            var nl = Environment.NewLine;
 
-        public ApplicationStartupException(string message, bool spotifyStatus) : base($"{CreateMessage(spotifyStatus)}\nAdditional details:\n{message}")
-        {
-        }
+            // Touching Spotify.Instance forces instance creation, but it fails if called too early
+            try
+            {
+                var tmp = Spotify.Instance;
+            }
+            catch (NullReferenceException)
+            {
+                return "Spotify instance status: not yet created.";
+            }
 
-        private static string CreateMessage(bool spotifyStatus = true)
-        {
             List<string> messages = new List<string>();
+            messages.Add("Spotify instance running status: " + Spotify.Instance.IsRunning);
 
-            if (spotifyStatus)
+            var status = Spotify.Instance.Status;
+            messages.Add("Spotify instance Status: ");
+            if (status == null)
+                messages.Add("null");
```

Comment:
This will end up on a separate line. Might be better to do something like this:

```C#
var status = ...
if (status == null)
    messages.Add("Spotify instance Status: null");
else
{
    messages.Add("Spotify instance Status:");
    ...
}
```

Reply:
Yep, was thinking about this solution as well.